File: HelperJobby/DTOs/UserJobInteractions/SavedJobDTO.cs
using HelperJobby.DTOs.Account;
using HelperJobby.DTOs.Job;

namespace HelperJobby.DTOs.UserJobInteractions;

public class SavedJobDTO
{
    public int JobId { get; set; }

    public int JobSeekerId { get; set; }

    public JobDTO Job { get; set; }

    public JobSeekerDTO JobSeeker { get; set; }

    public DateOnly DateSaved { get; set; }
}

File: HelperJobby/client-app/src/DTOs/userJobInteractionsDTOs/SavedJobDTO.ts
import {JobSeekerDTO} from "../accountDTOs/JobSeekerDTO";
import {JobDTO} from "../jobRelatetedDTOs/JobDTO";

export interface SavedJobDTO {
    jobId: number;
    jobSeekerId: number;
    job: JobDTO;
    jobSeeker: JobSeekerDTO;
    dateSaved: string;
}